packages/tools/viewer-alpha/test/apps/web/index.html

```diff
@@ -42,10 +42,10 @@
                 background-image: url("https://raw.githubusercontent.com/BabylonJS/Assets/master/textures/Checker_albedo.png");
             }
 
-             /* babylon-viewer-annotation:state(back-facing) {
+            /* babylon-viewer-annotation:state(back-facing) {

```

Comment:
wondering how much comment we should keep or if we need some kind of modes to run it ?

Reply:
There are buttons to test primary features, and for various other things I just have commented out code that is easy to uncomment for manual testing. I could add more buttons (or other UI) to the test app to toggle these things to test them, but I also don't want to over-invest in the test app...

Reply:
totally agree on the over invest part